pass-bakery-fe/src/components/BakeryMain.jsx

```diff
@@ -0,0 +1,19 @@
+import { menuData } from "../data/menuData";
+import TitleNav from "./TitleNav";
+import Menu from "./Menu";
+
+function BakeryMain(props) {
+  const menuObj = {};
+  menuData.menu.forEach((item, idx) => {
+    menuObj[menuData.menu[idx].name] = item;
```

Comment:
forEach modifies the original array and here it seems like you're creating a new object to print the menu item names. So map() can also be used which creates a new array and also returns empty array when there are no items. forEach returns undefined, check with menuData=[] and see what it returns.

Reply:
Thanks for pointing that out! I made a dummy variable and when I ran it, I got an error that it is not defined. I've also changed the forEach() to map().